controller/env/appenv.go

```diff
@@ -232,10 +233,10 @@ func (ae *AppEnv) FillRequestContext(rc *response.RequestContext) error {
 	if rc.SessionToken != "" {
 		_, err := uuid.Parse(rc.SessionToken)
 		if err != nil {
-			logger.WithError(err).Debug("failed to parse session id")
+			logger.WithError(err).Debug("failed to parse session traceId")
 			rc.SessionToken = ""
 		} else {
-			logger.Tracef("authorizing request using session id '%v'", rc.SessionToken)
+			logger.Tracef("authorizing request using session traceId '%v'", rc.SessionToken)
```

Comment:
"traceId" ?

Reply:
Goland rename went rouge. Fixing. It got a bunch of other ids in comments that I need to fix

Reply:
fixed